sidebar/auto-hide-sidebar-tst.css

```diff
@@ -0,0 +1,44 @@
+/*
+ * Description: A firefox userChrome.css modification to autohide sidebar when not in focus. It is based on the work of "TanzNukeTerror" and intended to be used with TreeStyleTabs.
+ *
+ * Screenshot: Original Theme - https://i.imgur.com/NKvSAYZ.jpg
+ *
+ * Contributor(s): img2tab, Chris Morgan, TanzNukeTerror, radamar
+ * 
+ */
+
+#sidebar {
+  max-width: none!important;
+  min-width: 0!important
+}
+
+#sidebar-box+#sidebar-splitter {
+  display: none!important
+}
+
+#sidebar-box[sidebarcommand=treestyletab_piro_sakura_ne_jp-sidebar-action] #sidebar-header {
```

Comment:
The autohide sidebar is very relevant and useful even without TST, and people using TST might want to hide the sidebar without removing the header. There is also an existing tweak to hide the header, so I'm not sure about the added value.

Could you remove the TST-specific lines? If you want to add them to sidebar/hide-sidebar-header.css
 with a comment explaining that you can use this instead to only hide the header for TST, I'll glady accept it.

Reply:
> Could you remove the TST-specific lines? 

sidebar-autohide now works for all sidebars.
> If you want to add them to sidebar/hide-sidebar-header.css with a comment explaining that you can use this instead to only hide the header for TST, I'll glady accept it.

done.